src/components/Layouts/Auth.tsx

```diff
@@ -59,7 +60,7 @@ export const Auth: React.FC<AuthProps> = ({
               </Box>
             </Paper>
             <Typography variant="body1" mt={4} pb={3}>
-              2023, DALGO ALL RIGHTS RESERVED
+              {moment().year()}, DALGO ALL RIGHTS RESERVED
             </Typography>
```

Comment:
Looks like this PR wont go in this weekend. Since we will be getting feedback from product team. 

Can we move this small change (right reserved 2024) to another PR ?

Reply:
yes , i have added it to other pr. I forgot to change branch. But that is taken care of.